prbench-rl/experiments/conf/agent/ppo_motion2d_0_passage.yaml

```diff
@@ -0,0 +1,17 @@
+# PPO agent configuration
+name: ppo
+
+# Core training parameters
+args:
+  total_timesteps: 1000000
+  num_eval_envs: 1
+  # Network architecture
+  hidden_size: 64
+
+# Training settings
+torch_deterministic: true
+cuda: false
+
+# Logging settings
+exp_name: "ppo_m2d_0_passage_train_1123_1m"
```

Comment:
I'm assuming that 1123 and 1119 etc. refers to the date. I wonder if we should remove this because we will probably rerun this at some later date and it might get confusing